CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # UNRELEASED
 
+**BREAKING CHANGES**
+
+-   Update matrix-rust-sdk to `9b05d0d82`, which includes:
+
+    -   `Device.requestVerification`, `UserIdentities.requestVerification`, and
+        `UserIdentities.verificationRequestContent` is not async anymore.
+        ([#3513](https://github.com/matrix-org/matrix-rust-sdk/pull/3513))
```

Comment:
Worth mentioning that converting an async function to not-async is not normally a breaking change in JS (though typescript might complain at you about the redundant `await`).

Reply:
Oh that's unexpected, thanks for letting me know.

Reply:
> Worth mentioning that converting an async function to not-async is not normally a breaking change in JS (though typescript might complain at you about the redundant await).

I disagree, as the method return value loses `.then` and `.catch` properties for non-async-await envs.